src/main/java/com/cloudbees/jenkins/support/configfiles/SecretHandler.java

```diff
@@ -87,4 +91,8 @@ public void characters(char[] ch, int start, int length) throws SAXException {
 
         return patchedFile;
     }
+
+    private static boolean isSecretBytes(String value) {
+        return (SecretBytes.decrypt(Base64.decode(value.substring(1, value.length() - 1))) != null);
```

Comment:
aren't they alwasy saved on the xml file with the toString() method? the toString method adds the `{}`